spec/factories/work_versions.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :work_version do
+    work
+    version_name { '1' }
+    aasm_state { 'draft' }
+    title { generate(:work_title) }
```

Comment:
@rschenk is this a FactoryBot thing? I was half-hoping for a custom method that creates a rando title via Faker. If you need inspiration https://github.com/psu-libraries/cho/blob/master/spec/support/metadata_factory.rb#L7

Reply:
This is a FactoryBot thing that generates a title that's guaranteed to be unique, but it's true that the titles will be super generic "Work Title 1", "Work Title 2" etc. I very much like them to be guaranteed to be unique for integration/feature tests, but I could combine the approaches...rando Faker title that includes a unique id somewhere in it